File: Python/8 kyu/Count of positives - sum of negatives.py
def count_positives_sum_negatives(arr):
    # your code here
    pos = 0
    neg = 0

    if not arr:
        return []

    for n in arr:
        if n > 0:
            pos += 1
        elif n < 0:
            neg += n

    return [pos, neg]

File: JavaScript/8 kyu/Count of positives - sum of negatives.js
function countPositivesSumNegatives(input) {
    // your code here
      if (!input || input.length === 0) { return [] };
    
      const posLength = input.filter(n => n > 0).length;
      const negSum = input.filter(n => n < 0).reduce((a,b) => a + b, 0);
      return [posLength, negSum];
    }